test/parallel/test-https-simple.js

```diff
@@ -36,14 +36,8 @@ const options = {
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
 };
 
-const tests = 2;
-let successful = 0;
-
 const testSucceeded = function() {
-  successful = successful + 1;
-  if (successful === tests) {
-    server.close();
-  }
+  server.close();
```

Comment:
I would prefer you change it to a function declaration (`function testSucceeded()` w/o the `const ... =`.) and move it to the bottom of the file. That should be more clear.